app/views/entities/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="myclass">
+  <%= link_to "Back", groups_path, class:"back-link" %>
+  <p>NEW ENTITY</p>
+  <div >
+  <%= link_to "Log Out", destroy_user_session_path,  class: "c-actions" %> 
+  </div>
+</div>
+
+<%= render "form", entity: @entity %>
+
+<br>
+
+<div>
+  <%= link_to "BACK TO ENTITIES", entities_path, class:"btn btn-light my-cstm-btn" %>
+</div>
```

Comment:
OPTIONAL

- [ ] Adding an entity is not a requirement. But it is giving an error. YOu can fix it.

![image](https://user-images.githubusercontent.com/38635158/176735316-075ce74f-2826-4937-8794-0e6a73c0c972.png)